src/modules/tests/tests-reducer-factory.ts

```diff
@@ -2,14 +2,19 @@
 import { TestCategory } from '../../shared/models/test-category';
 import { Action } from '@ngrx/store';
 import { StandardCarTestCATBSchema } from '@dvsa/mes-test-schema/categories/B';
+import { StandardTrailerTestCATBESchema } from '@dvsa/mes-test-schema/categories/BE';
 import { testsCatBReducer } from './tests.cat-b.reducer';
+import { testsCatBEReducer } from './tests.cat-be.reducer';
+import { TestResultUnionType } from './tests.model';
```

Comment:
Should use union in api definitions

Reply:
Will add it as the next PR when we use all the new schemas